parlai/crowdsourcing/tasks/turn_annotations/constants.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+AGENT_0 = 'Person1'
```

Comment:
Hmm - maybe add a TODO that these should be ported to Hydra at some point?

Reply:
I'm not sure `AGENT_0` and `AGENT_1` even need to be used anymore. The world can specify these but you'll never really need to be changing them. I guess you can change them if you _really_ wanted to give values, but I don't think they're important in the new setup.